src/jupyter_contrib_nbextensions/nbconvert_support/pre_codefolding.py

```diff
@@ -32,26 +32,52 @@ def fold_cell(self, cell, folded):
         """
         Remove folded lines and add a '<->' at the parent line
         """
+        # self.log.debug("CodeFoldingPreprocessor:: folding at: %s" % folded)
         lines = cell.splitlines(True)
 
         if folded[0] == 0 and (lines[0][0] == '#' or lines[0][0] == '%'):
             # fold whole cell when first line is a comment or magic
+            # self.log.debug("fold whole cell")
             return lines[0].rstrip('\n') + self.fold_mark + '\n'
-        fold_indent = 0
+        
+        foldIndent = 0
         fold = False
         fcell = ""
+        isSkipLine  = False
+        skipped = ""
+
         for i, l in enumerate(lines):
             # fold indent level
-            indent = len(l) - len(l.lstrip(' '))
-            if indent <= fold_indent:
+            lstrip    = l.lstrip(r' \t') # strip tabs and spaces
+            indent    = len(l) - len(lstrip)
+            isSkipLine = lstrip[0] == "#" or lstrip in ["\n"] # is it a comment or an empty line
```

Comment:
by the way, how do you reproduce the linter errors locally?
I am using atom, but flake8 is giving me not the same errors? Do you have a solution for this or you just look at travis?